src/Corrade/Utility/Debug.cpp

```diff
@@ -59,13 +108,16 @@ template<Debug::Color c> Debug::Modifier Debug::colorInternal() {
         if(!noSpaceBefore) debug << Debug::nospace;
         debug << code;
         if(noSpaceBefore) debug << Debug::nospace;
```

Comment:
I think putting space at proper place before/after space is important also for the Windows implementation to avoid visual artifacts.

Similarly for the bold implementation below.